deduplication/src/main/java/gov/cdc/nbs/deduplication/seed/JobConfig.java

```diff
@@ -45,7 +45,7 @@ public JobConfig(
   @Bean("readNbsWriteToMpi")
   public Step step1(JobRepository jobRepository, PlatformTransactionManager transactionManager) {
     return new StepBuilder("Read and transform NBS data", jobRepository)
-        .<NbsPerson, Cluster>chunk(10, transactionManager)
+        .<NbsPerson, Cluster>chunk(100, transactionManager)
```

Comment:
Suggestion: Would you mind updating this as well as the next chunk size to be configurable with defaults of what you have?